tests/web/test_assets.py

```diff
@@ -81,3 +83,62 @@ def test_get_assets_limit(client, b):
         res = client.get(ASSETS_ENDPOINT + '?search=abc&limit=1')
         assert res.status_code == 200
         assert len(res.json) == 1
+
+
+@pytest.mark.bdb
+@pytest.mark.tendermint
+def test_get_assets_tmint(client, tb):
+    from bigchaindb.models import Transaction
+    from bigchaindb.backend.localmongodb.connection import LocalMongoDBConnection
+
+    if isinstance(tb.connection, LocalMongoDBConnection):
```

Comment:
I'm a bit confused, what's the reason to mark the test for `mdb` if we don't test it actually against the `mongodb` backend? I'd probably just remove `pytest.mark.bdb` **or** (it's even better, probably) I'd be totally fine in duplicating the test (yep, copypasta) so we have one for `mdb` and one for `tendermint`. Repetition is fine in tests :)

(same applies for the other test below)

Reply:
In case we merge these changes back into `master`, the tendermint tests won't affect anything when tests are run using `mongodb` or `rethinkdb`

Reply:
Oh I see, what if we use [`pytest.mark.skipif`](https://docs.pytest.org/en/latest/skipping.html#id1)?